From 28728a50b461475c1833dbed995a389733439a66 Mon Sep 17 00:00:00 2001
From: Erwan Le Ray <erwan.leray@foss.st.com>
Date: Fri, 27 Mar 2020 15:16:23 +0100
Subject: [PATCH 125/529] ARM: dts: stm32: activate usart3 and uart7 for
 stm32mp157x-dkx a7-examples

Activate usart3 (available on GPIO expansion connector) and uart7
(available on Arduino connector) for stm32mp157x-dkx a7-examples.

Signed-off-by: Erwan Le Ray <erwan.leray@foss.st.com>
Change-Id: Id1d8660ada95c90a3c67e0ea24a0d200d07592f9
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/277717
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/277718
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/277719
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/277720
Reviewed-by: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Valentin CARON <valentin.caron@foss.st.com>
Reviewed-by: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
Tested-by: Valentin CARON <valentin.caron@foss.st.com>
Domain-Review: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
---
 arch/arm/boot/dts/stm32mp157a-dk1-a7-examples.dts | 8 ++++++++
 arch/arm/boot/dts/stm32mp157c-dk2-a7-examples.dts | 8 ++++++++
 arch/arm/boot/dts/stm32mp157d-dk1-a7-examples.dts | 8 ++++++++
 arch/arm/boot/dts/stm32mp157f-dk2-a7-examples.dts | 8 ++++++++
 4 files changed, 32 insertions(+)

--- a/arch/arm/boot/dts/stm32mp157a-dk1-a7-examples.dts
+++ b/arch/arm/boot/dts/stm32mp157a-dk1-a7-examples.dts
@@ -12,3 +12,11 @@
 	model = "STMicroelectronics STM32MP157A-DK1 configured to run Linux A7 examples";
 	compatible = "st,stm32mp157a-dk1-a7-examples", "st,stm32mp157a-dk1", "st,stm32mp157";
 };
+
+&uart7 {
+	status = "okay";
+};
+
+&usart3 {
+	status = "okay";
+};
--- a/arch/arm/boot/dts/stm32mp157c-dk2-a7-examples.dts
+++ b/arch/arm/boot/dts/stm32mp157c-dk2-a7-examples.dts
@@ -12,3 +12,11 @@
 	model = "STMicroelectronics STM32MP157C-DK2 configured to run Linux A7 examples";
 	compatible = "st,stm32mp157c-dk2-a7-examples", "st,stm32mp157c-dk2", "st,stm32mp157";
 };
+
+&uart7 {
+	status = "okay";
+};
+
+&usart3 {
+	status = "okay";
+};
--- a/arch/arm/boot/dts/stm32mp157d-dk1-a7-examples.dts
+++ b/arch/arm/boot/dts/stm32mp157d-dk1-a7-examples.dts
@@ -12,3 +12,11 @@
 	model = "STMicroelectronics STM32MP157D-DK1 configured to run Linux A7 examples";
 	compatible = "st,stm32mp157d-dk1-a7-examples", "st,stm32mp157d-dk1", "st,stm32mp157";
 };
+
+&uart7 {
+	status = "okay";
+};
+
+&usart3 {
+	status = "okay";
+};
--- a/arch/arm/boot/dts/stm32mp157f-dk2-a7-examples.dts
+++ b/arch/arm/boot/dts/stm32mp157f-dk2-a7-examples.dts
@@ -12,3 +12,11 @@
 	model = "STMicroelectronics STM32MP157F-DK2 configured to run Linux A7 examples";
 	compatible = "st,stm32mp157f-dk2-a7-examples", "st,stm32mp157f-dk2", "st,stm32mp157";
 };
+
+&uart7 {
+	status = "okay";
+};
+
+&usart3 {
+	status = "okay";
+};
